.github/workflow-scripts/gh-pages/deploy.js

```diff
@@ -69,24 +69,27 @@ async function deploy(branch, docsPath, exec) {
     await exec.exec('git', ['checkout', 'gh-pages']);
 
     console.info('ℹ️ Applying the stash with the storybook changes');
-    await exec.exec('git', ['stash', 'pop']);
+    // force the stash to be applied
+    await exec.exec('git', ['checkout', 'stash', '--', '.']);
```

Comment:
just trying to get my head around this.... you already did a checkout of gh-pages right? what exactly is this line doing? :-)

Reply:
This is to force apply the stash. Apparently there is no such thing as `git stash pop --force` 🤔  This will apply the stash over whatever files already exist.